chat-widget/src/plugins/createChatTranscript.ts

```diff
@@ -0,0 +1,493 @@
+/* eslint-disable no-useless-escape */
+
+class TranscriptHTMLBuilder {
+    private options: any; // eslint-disable-line @typescript-eslint/no-explicit-any

```

Comment:
Im not fan of having this constants here, I will always favor to use external components where in case of need to update the message doesnt involves to touch the code with the logic,  plust it seems some texts are duplicated from dummyDefaultProps...

Reply:
`dummyDefaultProps` is a file used to display all the default props verbosely. Otherwise, the default strings are only used in `TranscriptHTMLBuilder`.